nautobot/extras/api/nested_serializers.py

```diff
@@ -138,3 +139,29 @@ class NestedRelationshipAssociationSerializer(WritableNestedSerializer):
     class Meta:
         model = models.RelationshipAssociation
         fields = ["id", "url", "relationship", "source_id", "destination_id"]
+
+
+class NestedJobScheduleSerializer(serializers.Serializer):
```

Comment:
Should this be a `WritableNestedSerializer` subclass?

Reply:
As of now, the schedule is either read-only or is required to be filled. As I understand it, the purpose of `WritableNestedSerializer` is to allow for primary key values as well? If so, I don’t think this would currently make sense.